accounts/models.py

```diff
@@ -95,7 +97,7 @@ class User(django.contrib.auth.models.AbstractBaseUser,
     email = django.db.models.EmailField(_('Email'), unique=True)
     timezone = timezone_field.TimeZoneField(default='America/New_York')
 
-    company = django.db.models.ForeignKey(Company, null=True, blank=True, related_name='users')
+    company = django.db.models.ForeignKey(Company, null=True, related_name='%(app_label)s_%(class)s_users')
```

Comment:
@awartani It is included but have `null=True`, so whoever don't want to use it they can just ignore it and remove it from Admin site. However, I think `company` is a general purpose field.